cloud-providers/azure/kubernetes/payara-micro-distributed-cache/src/main/java/be/rubus/accelerator/k8s/azure/CounterService.java

```diff
@@ -0,0 +1,23 @@
+package be.rubus.accelerator.k8s.azure;
```

Comment:
Why is this the package name? If it's a Payara Example shouldn't it be `fish.payara.example.something`?

Reply:
You're right, @MattGill98, I'll fix that.

Reply:
Done, @MattGill98 , can you review?